src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
If we go with this option, given `--config=my.txt` - should `pydocstyle` look for `my.txt` files all the way up, ignoring e.g., `tox.ini` files, or _start_ with _this specific_ `my.text` file and look up for "normal" config files? I tend toward the latter. In any case, whatever is decided should be well documented, as this is starting to get confusing.